prophy/composite.py

```diff
@@ -584,3 +594,12 @@ def __init__(cls, name, bases, attrs):
             extend_union_descriptor(cls, descriptor)
             add_union_properties(cls, descriptor)
         super(union_generator, cls).__init__(name, bases, attrs)
+
+
+
+class Descriptor(object):
```

Comment:
"Descriptor" is name used for class or union layout (I mean struct._descriptor field).
When we mean "the descriptor of single field", I think we should disambiguate and say: "FieldDesc" or "FieldDescriptor".